docs/book/user-guide/llmops-guide/evaluation/evaluation.md

```diff
@@ -0,0 +1,54 @@
+---
+description: Track how your RAG pipeline improves using evaluation and metrics.
+---
+
+# Evaluation and metrics for LLMOps
+
+In this section, we'll explore how to evaluate the performance of your RAG
+pipeline using metrics and visualizations. Evaluating your RAG pipeline is
+crucial to understanding how well it performs and identifying areas for
+improvement. With language models in particular, it's hard to evaluate their
+performance using traditional metrics like accuracy, precision, and recall. This
+is because language models generate text, which is inherently subjective and
+difficult to evaluate quantitatively.
+
+Our RAG pipeline is a whole system, moreover, not just a model, and evaluating
+it requires a holistic approach. We'll look at various ways to evaluate the
+performance of your RAG pipeline but the two main areas we'll focus on are:
+
+- [Retrieval evaluation](retrieval.md), so checking that the
+  retrieved documents or document chunks are relevant to the query.
+- [Generation evaluation](generation.md), so checking that the
+  generated text is coherent and helpful for our specific use case.
+
+In the previous section we built out a basic RAG pipeline for our documentation
+question-and-answer use case. We'll use this pipeline to demonstrate how to
+evaluate the performance of your RAG pipeline. If you were running this in a
+real setting, you might want to set up evaluation to check the performance of a
+raw LLM model (i.e. without any retrieval / RAG components) as a baseline, and
+then compare this to the performance of your RAG pipeline. This will help you
+understand how much value the retrieval and generation components are adding to
+your system. We won't cover this here, but it's a good practice to keep in mind.
```

Comment:
As this is more of a sidenote, we can parse it into a "callout" with an info label